src/views/settings/components/settings.js

```diff
@@ -0,0 +1,189 @@
+import React, { PropTypes, Component } from 'react';
+import {
+     Panel,
+} from 'react-bootstrap/lib/';
+import PasswordDialog from '~/src/views/settings/components/password';
+import ColorPickerDialog from '~/src/views/settings/components/backgroundColor';
+import RefreshAPI from '~/src/views/settings/components/apiToken';
+import Statistic from '~/src/views/settings/components/statistic';
+import ItemTypes from '~/src/views/settings/components/itemTypes';
+import Logo from '~/src/assets/images/logo.png';
+import './settings.scss';
+
+export default class Settings extends Component {
+
+    static propTypes = {
+        showModal: PropTypes.boolean,
```

Comment:
this should be `PropTypes.bool` not `boolean`. This is showing errors in the console. Make sure to change this in all of the other settings components